activestorage/README.md

```diff
@@ -106,6 +106,25 @@ Variation of image attachment:
 <%= image_tag user.avatar.variant(resize_to_limit: [100, 100]) %>
 ```
 
+## Delivery Routes
+Active storage provides two primary ways to route assets.
```

Comment:
“Assets” is potentially confusing here since we use it to mean other things elsewhere in Rails. “Files”?